### ISSUE TITLE
automatically send all ORPHAN sources with RB > 0.67 to possible

### ISSUE THREAD

**Issue by djones1040** [genghisken/ps1#169]
Hi Ken, the team vetting YSE sources have asked if it's possible to send all ORPHAN sources with RB > 0.67 to the possible list directly?

**Comment by genghisken** [comment]
This might be an easy one...  The process of sending ORPHANs to Possible is easily done by users.  There are a few features in the web pages that you don't know about!  You can select by object type very easily thus:

https://star.pst.qub.ac.uk/sne/ps1yse/psdb/followup_quickview/4/?sherlockClassification=ORPHAN

Got more than 100 (default pagination)?

https://star.pst.qub.ac.uk/sne/ps1yse/psdb/followup_quickview/4/?sherlockClassification=ORPHAN&nobjects=1000

Now you can check the letter "P" in the HEADER and all will be highlighted as "P"ossible.  Scroll down and click the Classify button.  Bob's your uncle, they all get moved to possible.  (Page might time out, so give it a minute.)

Doing this automatically is also possible - it's slightly more involved than changing the list number, because the objects get given internal names.  I'm guessing you want to send them there once the images have arrived and ML has run (and because they get left in the eyeball list).

**Comment by djones1040** [comment]
thanks Ken, this will work great - but don't suppose you have a similar filter for RB factor? like https://star.pst.qub.ac.uk/sne/ps1yse/psdb/followup_quickview/4/?sherlockClassification=ORPHAN&rbfactor_gt=0.67

**Comment by genghisken** [comment]
No, not yet, but I've been looking at doing this very thing, so I'll be implementing this very soon (within days I hope).

**Comment by genghisken** [comment]
So much for implementation within days!!  This is (finally) implemented now for Old Quickview only at the moment.

Use two underscores (it's a lazy Django-ism) for less than (__lt), less than or equal to (__lte), greater than (__gt), greater than or equal to (__gte).  For historical reasons the RB Factor parameter is called "confidence_factor", so in Old Quickview you can have:

https://star.pst.qub.ac.uk/sne/ps1yse/psdb/followup_quickview/3/?sherlockClassification=ORPHAN&confidence_factor__gt=0.67

https://star.pst.qub.ac.uk/sne/ps1yse/psdb/followup_quickview/3/?sherlockClassification=ORPHAN&confidence_factor__gte=0.67

https://star.pst.qub.ac.uk/sne/ps1yse/psdb/followup_quickview/3/?sherlockClassification=ORPHAN&confidence_factor__lt=0.67

https://star.pst.qub.ac.uk/sne/ps1yse/psdb/followup_quickview/3/?sherlockClassification=ORPHAN&confidence_factor__lte=0.67

The same works for "followup_flag_date" (where "equal to" makes more sense!)
E.g.
https://star.pst.qub.ac.uk/sne/ps1yse/psdb/followup_quickview/3/?sherlockClassification=ORPHAN&confidence_factor__gte=0.67&followup_flag_date__gte=2020-10-11

**Comment by djones1040** [comment]
awesome, thanks Ken!  Definitely familiar with the weird Django filter language.  I'll pass this along to the group.  Could close the issue if you want, or leave it open until you've had a chance to finish the non-quickview pages?
